js/gateio.js

```diff
@@ -27,6 +27,7 @@ module.exports = class gateio extends Exchange {
                 'fetchOrderTrades': true,
                 'fetchOrders': true,
                 'fetchOrder': true,
+                'fetchMyTrades': true,
```

Comment:
We can't really set it to `true` without adding an implementation for it (see how it is implemented in other exchanges). Would you want to give it try?

Reply:
I wrote fetchMyTrades a while back, must have forgot the metadata :)

Reply:
@frosty00 lol, i'm a blind idiot then ) Thx for pointing that out! )

@ndubel pardon for the confusion!

Reply:
@frosty00 we've sent you a message to your email address (the address we got from you earlier), check it out when you have time for it) or let us know if you haven't received anything.